weatherv3/www/js/controllers.js

```diff
@@ -0,0 +1,116 @@
+angular.module('starter.controllers', ['ionic'])
+
+.constant('FORECASTIO_KEY', '76f3ef0ca4c614a2ced71001eb970aa8')
+
+
+.controller('HomeCtrl', function($scope,$state,Weather,DataStore) {
+    //read default settings into scope
+    console.log('inside home');
+    $scope.city  = DataStore.city;
+    var latitude  =  DataStore.latitude;
+    var longitude = DataStore.longitude;
+
+    //some code found here: https://www.binpress.com/tutorial/a-simple-weather-forecast-app-with-ionic-framework-and-forecastio/107
+    Weather.getCurrentWeather(latitude,longitude).then(function(resp) {
+      var current = resp.data;
+      $scope.temperature = Math.round(current.currently.temperature) + "°F";
+      $scope.apparentTemperature = Math.round(current.currently.apparentTemperature) + "°F";
+      $scope.summary = current.currently.summary.toLowerCase();
+      $scope.tempDif = Math.abs(Math.round(current.currently.temperature) - Math.round(current.currently.apparentTemperature));
+      $scope.windSpeed = Math.round(current.currently.windSpeed);
+      var intensity = current.currently.precipIntensity;
+
```

Comment:
I think that it does work. It was a bit hard to test but I haven't seen it fail.